From: Leon Romanovsky <leonro@mellanox.com>
Date: Wed, 2 May 2018 13:15:24 +0300
Subject: RDMA/hns: Drop local zgid in favor of core defined variable
Patch-mainline: v4.18-rc1
Git-commit: ed3dd9b017b85e00a459c35bd4d3fe2b83b0d092
References: bsc#1104427 FATE#326416

The zgid is already provided by IB/core, so there is no need in locally
defined variable, let's drop it and reuse common one.

Signed-off-by: Leon Romanovsky <leonro@mellanox.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/hns/hns_roce_main.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/infiniband/hw/hns/hns_roce_main.c
+++ b/drivers/infiniband/hw/hns/hns_roce_main.c
@@ -101,7 +101,6 @@ static int hns_roce_del_gid(struct ib_de
 {
 	struct hns_roce_dev *hr_dev = to_hr_dev(device);
 	struct ib_gid_attr zattr = { };
-	union ib_gid zgid = { {0} };
 	u8 port = port_num - 1;
 	unsigned long flags;
 	int ret;
